app_vdi.py

```diff
@@ -0,0 +1,212 @@
+        dcc.Store(
+            id='data-store',
+            storage_type='session',
+            data=VDI_DATA.copy()
+        ),
+        html.Div(
+            id='top-panel-div',
+            className='app__container',
+            children=[
+                html.Div(
+                    id='timeseries-div',
+                    className='app__timeseries',
+                    title='Hover description',
+                    children=dcc.Graph(
+                        id='timeseries-plot',
+                        figure=go.Figure(data=[go.Scatter(x=[], y=[])]),
+                        style={'width': '70%', 'height': '90%'}
+                    ),
+                ),
+                html.Div(
+                    id='load-data-div',
+                    className='app__load_data',
+                    title='Hover description',
+                    children=[
+                        dcc.Upload(
+                            id='load-data',
+                            className='app__upload',
+                            children=html.Div([
+                                'Drag and Drop or ',
+                                html.A('Select Files')
+                            ]),
+                            multiple=False,
+                            style={
+                                'width': '100%',
+                                'height': '60px',
+                                'lineHeight': '60px',
+                                'borderWidth': '1px',
+                                'borderStyle': 'dashed',
+                                'borderRadius': '5px',
+                                'textAlign': 'center',
+                                'margin': '10px'
+                            },
+                        ),
+                        html.Button(
+                            id='run-btn',
+                            className='app__button',
+                            children='Run'
+                        )
+                    ]
+                )
+            ]
+        ),
+        html.Div(
+            id='bottom-panel-div',
+            className='app__container',
+            children=[
+                html.Div(
+                    id='parameters-div',
+                    className='app__parameters',
+                    title='Parameters description',
+                    children=[
+                        html.Div(
+                            id='param-tech-div',
+                            className='app__parameters__section',
+                            children=[
+                                html.Div('Technische Parameter'),
+                                html_param_input('Zyklenwirkungsgrad', 76.6, '%'),
+                                html_param_input('Entladetiefe', 37, '%'),
+                                html_param_input('C-Rate', 0.5),
+                            ]
+                        ),
+                        html.Div('Investitionskosten-div'),
```

Comment:
@robmip - I leave it for you to fill the different sections, don't focus on the styling at first. this can all be done in the end :)